smithy-aws-protocol-tests/model/shared-types.smithy

```diff
@@ -121,10 +121,6 @@ list LongSet {
     member: Long,
 }
 
-list TimestampList {
```

Comment:
This was a duplicate definition that was also cleaned up in the `smithy-protocol-tests` copy.